lib/chai/utils/overwriteProperty.js

```diff
@@ -4,7 +4,9 @@
  * MIT Licensed
  */
 
+var chai = require('../../chai');
```

Comment:
Currently `addMethod` and `addProperty` have this too and there's been no issue reported about it. However, I'd like to have more time to check this before giving a confident response.